scripts/update-sbom.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$(dirname ${BASH_SOURCE[0]})
+ROOT_DIR=$(realpath "${SCRIPT_DIR}/../")
+PURLS_FILE="${ROOT_DIR}/purls.txt"
+
+LIBMONGOC_VERSION=$(cat ${ROOT_DIR}/src/LIBMONGOC_VERSION_CURRENT | tr -d '[:space:]')
+LIBMONGOCRYPT_VERSION=$(cat ${ROOT_DIR}/src/LIBMONGOCRYPT_VERSION_CURRENT | tr -d '[:space:]')
```

Comment:
I assume we don't need to call out that these files must be updated since the SBOM generation step follows the "Ensure version information is correct" step, which updates these.